coalib/results/result_actions/GeneratePatchesAction.py

```diff
@@ -0,0 +1,89 @@
+from pygments.lexers import guess_lexer_for_filename
+from pyprint.ConsolePrinter import ConsolePrinter
+
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.output.printers.LogPrinter import LogPrinter
+from coalib.parsing.FilterHelper import FilterHelper
+from coalib.coala_modes import mode_normal
+from coalib.parsing.DefaultArgParser import default_arg_parser
+
+
+OBJECT_INDEX = FILENAME_INDEX = 0
+DEFAULTBEAR = 'SpaceConsistencyBear'
+
+
+def show_possibilities(console_printer, i, action):
+    console_printer.print('[{:>4}]  {}. Apply patch (\'{}\')'.format('', i,
+                                                                     action))
+
+
+def create_arg_parser(files, bears):
+    """
+    A function that generates a `default_arg_parser`
+
+    :param files: A list that contains filenames.
+    :param bears: A list that contains name of bears.
+    :return:      An object of type `default_arg_parser`.
+    """
+    args = default_arg_parser().parse_args()
+    args.files = files
+    args.bears = bears
+    args.default_actions = '*: ApplyPatchAction'
```

Comment:
`ApplyPatchAction` -> `GeneratePatchAction`

Reply:
you want to apply the patch, not to generate another one :)